CMakeLists.txt

```diff
@@ -94,7 +94,13 @@ else()
   if(WITH_ZLIB)
     find_package(zlib REQUIRED)
     add_definitions(-DZLIB)
-    include_directories(${ZLIB_INCLUDE_DIR})
+    if (DEFINED ZLIB_INCLUDE_DIRS)
```

Comment:
please note cmake also consider a condition also false if `ZLIB_INCLUDE_DIRS` is not defined. so for simplicity and consistency, i'm inclined to just put  `if(ZLIB_INCLUDE_DIRS)`. and to be more consistent with other part of the cmake scripts in this project, could you remove the space after `if`?

Reply:
Good to know that!